### PR TITLE
Microverse Project 3: Newsweek page clone. 

### PR DESCRIPTION
**Pull request by franklinben23**
For this project, we tried to replicate the Newsweek web site as close as we could, the tools we used for the task were Html, CSS, and the bootstrap front end framework. made collaboratively.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2039 @@
...
+          <p>© 2020 NEWSWEEK DIGITAL LLC</p>
+        </div>
+        <div class="icons">
+          <i class="fa fa-facebook-square"></i>
+          <i class="fa fa-twitter"></i>
+          <i class="fa fa-instagram"></i>
+          <i class="fa fa-tumblr"></i>
+          <i class="fa fa-linkedin"></i>
+        </div>
+      </div>
```
Comment (peterrobert) [inline index.html:2011]: There spaces between the icons. You can use the bootstrap padding class to add this styling. Follow [this](https://getbootstrap.com/docs/4.0/utilities/spacing/) to learn more.

**YOURS**
yoursIcons

**ORIGINAL**
icons original

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2039 @@
...
+          </div>
+          <div class="hollow-tag">
+            <p>CULTURE</p>
+          </div>
+        </div>
+        <a class="feature-link" href="#"
+          >Banning Porn Is a Bad Idea. There Are Other Ways To Improve the
+          Internet</a
+        >
+      </div>
```
Comment (peterrobert) [inline index.html:1328]: There should be spacing between the articles. Use bootstrap spacing classes for this. Kindly check below.
Follow [this](https://getbootstrap.com/docs/4.0/utilities/spacing/) link to learn more on bootstrap spacing.

**yours**
spacing yours
**original**
spacing original

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2039 @@
...
+        <li>Weather</li>
+        <li>
+          <div class="input-group form-nav flex-nowrap align-items-center">
+            <input type="text" class="form-control" placeholder="search" />
+            <div class="input-group-append">
+              <button class="btn p-0" type="button">
+                <i class="fa fa-search" aria-hidden="true"></i>
+              </button>
+            </div>
+          </div>
```
Comment (peterrobert) [inline index.html:71]: Your search bar does not resemble the original. Kindly make your search bar resemble the original.
**yours**
search yours
**original**
search original

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @franklinben23  and @charlyeneh 👋🏿  very nice project 👍🏿 you have done a good job. I love the way you have used bootstrap classes just like it was required to keep up the good work.

However, there are a few changes to be made: kindly check below:

### if you have any question kindly reach me on slack @peterrobert

### HAPPY CODING 💻

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @franklinben23 and @charlyeneh  :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by charlyeneh** [comment]
Thank you very much for your preview and corrections! They were very insightful.

**Pull request merged**